src/main/java/seedu/bankwithus/AccountList.java

```diff
@@ -35,13 +37,13 @@ public AccountList(BankWithUs bwu) {
      * @param bwu     the main bankWithUs program
      */
     public AccountList(Scanner scanner, BankWithUs bwu) {
-        accounts = new ArrayList<Account>();
+        accounts = new ArrayList<>();
         this.ui = bwu.getUi();
         Parser parser = new Parser(this);
         try {
             parser.parseSavedFile(scanner);
         } catch (Exception e) {
-            if (e.equals(new SaveFileIsEmpty())) {
```

Comment:
For exceptions, use catch blocks to differentiate between whoch exceptions to catch. E.g. 
try {...} 
catch (NumberFormatException e) {...}
catch (EmptySaveFileException e) {...}